### PR TITLE
fix sample name for abricate output (#2576) and update to 0.9.3

### PR DESCRIPTION
**Pull request by thanhleviet**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/abricate/abricate.xml [hunk 1]
```diff
@@ -9,7 +9,12 @@
     <expand macro="version_command" />
```
Comment (bgruening) [inline tools/abricate/abricate.xml]: @thanhleviet thanks! Please also increase the tool version number.

File: tools/abricate/test-data/output_minid100.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
-#FILE	SEQUENCE	START	END	GENE	COVERAGE	COVERAGE_MAP	GAPS	%COVERAGE	%IDENTITY	DATABASE	ACCESSION
-NC_002952.2	44919	46928	mecA_15	1-2010/2010	===============	0/0	100.00	100.00	resfinder	AB505628
-NC_002952.2	55890	56621	erm(A)_1	1-732/732	===============	0/0	100.00	100.00	resfinder	X03216
-NC_002952.2	56747	57529	spc_1	1-783/783	===============	0/0	100.00	100.00	resfinder	X02588
-NC_002952.2	1796513	1797244	erm(A)_1	1-732/732	===============	0/0	100.00	100.00	resfinder	X03216
-NC_002952.2	1797370	1798152	spc_1	1-783/783	===============	0/0	100.00	100.00	resfinder	X02588
-NC_002952.2	1913827	1914672	blaZ_36	1-846/846	===============	0/0	100.00	100.00	resfinder	AJ400722
+#FILE	SEQUENCE	START	END	GENE	COVERAGE	COVERAGE_MAP	GAPS	%COVERAGE	%IDENTITY	DATABASE	ACCESSION	PRODUCT
+MRSA0252.fna	NC_002952.2	40786	41556	aadD_2	1-771/771	===============	0/0	100.00	100.00	resfinder	M19465_dupe	resfinder~~~aadD_2~~~M19465_dupe Alternate name; ant(4')-Ia and aadD2
```
Comment (bgruening) [inline tools/abricate/test-data/output_minid100.txt]: Do you know what `~~~aadD_2~~~` is? This looks wired, why does this change? Any idea are you familiar with this tool?

Comment (thanhleviet) [reply tools/abricate/test-data/output_minid100.txt]: That's the description format for each gene as described here https://github.com/tseemann/abricate#making-your-own-database
`>DB~~~ID~~~ACC DESC`
I don't know when this syntax has been changed, likely recently.

Comment (thanhleviet) [reply tools/abricate/test-data/output_minid100.txt]: @Slugger70 any thoughts? can it be merged otherwise?

Comment (Slugger70) [reply tools/abricate/test-data/output_minid100.txt]: Torsten uses ~~~ as a delimiter as it's unlike anything else. 
It should be ok.

Comment (Slugger70) [reply tools/abricate/test-data/output_minid100.txt]: He just added some more information to the database field.

**Pull request merged**